src/Symfony/Component/HttpKernel/Attribute/ValueResolver.php

```diff
@@ -17,10 +17,10 @@
 class ValueResolver
 {
     /**
-     * @param class-string<ValueResolverInterface>|string $name
+     * @param class-string<ValueResolverInterface>|string $resolver
      */
     public function __construct(
-        public string $name,
+        public string $resolver,
```

Comment:
See https://github.com/symfony/symfony/pull/48992#discussion_r1126468048

Reply:
Should it be reflected in `AsPinnedValueResolver`?